ui-components/src/navigation/LanguageNav.tsx

```diff
@@ -8,20 +8,25 @@ export type LangItem = {
   label: string
 }
 
+export interface LanguageNavLang {
+  list: LangItem[]
+  codes: string[]
+}
+
```

Comment:
Would it be useful to have a map of the languages indexed by the code? In looking at some of the logic around checking the list and things like activeLangPath, it may be more efficient to have it like:
```
enum LanguageEnum {
  en
  es
}
// added to LanguageNavLang interface
map: { [key: LanguageEnum]: string }

// so the map itself would be like
const languageMap = {
  en: "English",
  es: "Español"
}
```

Reply:
I see, @dominikx96 and @pbn4 , when we have time to implement a county table, what do you think about moving languages out of the environment variables?

Reply:
@seanmalbert Lets capture this as part of work on listings management